addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15371,13 +15374,13 @@ msgstr ""
 #. Description of settings category "Services -> Zeroconf" with label #1259
 #: system/settings/settings.xml
 msgctxt "#36341"
-msgid "Category containing settings for how the zeroconf network discovery service is handled, required for AirPlay."
+msgid "Category containing settings for how the \"Zeroconf\" network discovery service is handled, required for AirPlay."
 msgstr ""
 
 #. Description of setting "Services -> Zeroconf -> Announce services to other systems"  with label #1260
 #: system/settings/settings.xml
 msgctxt "#36342"
-msgid "Allows applications on the network to discover enabled services via Zeroconf."
+msgid "Allows applications on the network to discover enabled services via \"Zeroconf\"."
```

Comment:
Why have you put it in quotes everywhere? Again this IMO is not intended as internal cross reference but rather just the name of the tool used.